### ISSUE TITLE
Theme in `next` branch gives compatibility warning

### ISSUE THREAD

**Issue by Ghostbird** [apache/incubator-echarts#13519]
### Version
5.0.0-beta.2

### Steps to reproduce
Render an echarts graph with the dark theme

### What is expected?
No warnings

### What is actually happening?
> [ECharts] DEPRECATED: textStyle hierarchy in label has been removed since 4.0. All textStyle properties are configured in label directly now.

---
https://github.com/apache/incubator-echarts/blob/cb5cfdc0682402380824ba9756bf116b3b945d14/theme/dark.js#L174

This line uses `textStyle`, oddly this change was introduced in the `next` branch where this functionality is deprecated.
It should say:
```
label: {
  color: contrastColor
}
```

**Comment by plainheart** [comment]
Perhaps, we should update the deprecated usage in each theme file later.

**Comment by Ghostbird** [comment]
The main thing that threw me off, was that these changes are not in `master`. So why were they added in the `next` branch? Or is this already fixed in `master` and will it be merged once `next` is ready?

**Comment by plainheart** [comment]
The `next` branch is our recent active developing branch. Some new changes in themes will be made to it. Before publishing the stable release, the new changes will be merged into the `master` branch.
